CONTRIBUTING.md

```diff
@@ -114,18 +114,23 @@ When making a contribution, adhering to the [GitHub flow](https://guides.github.
 
    ```bash
    # Clone your fork of the repo into the current directory
-   git clone https://github.com/<your-username>/<repo-name>
+   git clone https://github.com/<your-username>/Scribe-iOS.git
    # Navigate to the newly cloned directory
-   cd <repo-name>
+   cd Scribe-iOS.git
    # Assign the original repo to a remote called "upstream"
-   git remote add upstream https://github.com/<upsteam-owner>/<repo-name>
+   git remote add upstream https://github.com/scribe-org/Scribe-iOS.git
    ```
+   Now, if you run `git remote -v` you should see two remote repositories named:
+
+   - `origin` (forked repository)
+   - `upstream` (Scribe-iOS repository)
+
 
 2. If you cloned a while ago, get the latest changes from upstream:
 
    ```bash
-   git checkout <dev-branch>
-   git pull upstream <dev-branch>
+   git checkout main
```

Comment:
These here we can leave as `<dev-branch>` as was done in the [PR we're referencing] :)